go/interpreter/lfvm/instructions_test.go

```diff
@@ -666,83 +666,46 @@ func TestTransientStorageOperations(t *testing.T) {
 	}
 }
 
-func TestExpansionCostOverflow(t *testing.T) {
-	memTestValues := []uint64{
-		maxMemoryExpansionSize,
-		maxMemoryExpansionSize + 1,
-		math.MaxUint64,
-	}
+func TestGenericDataCopy_ReturnsErrorOn(t *testing.T) {
```

Comment:
Is there a positive test making sure that data is copied correctly if all conditions are satisfied?